app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -15,6 +16,7 @@ import ProjectBoardPropTypes from '../shapes/projectBoard';
 import Notifications from '../Notifications';
 import { removeNotification } from '../../actions/notifications';
 import StorySearch from '../search/StorySearch';
+import {SprintButton} from '../sprint/SprintButton';
```

Comment:
Add spaces between `{}` 💅 
```suggestion
import { SprintButton } from '../sprint/SprintButton';
```